libgloss/patmos/close.c

```diff
@@ -33,11 +33,20 @@
 #undef errno
 extern int  errno;
 
+//******************************************************************************
+/// patmos-plug: close: stub called if 'patmosplug_close(int)' is not defined.
```

Comment:
Could use some more explanation, perhaps:

```suggestion
/// patmos-plug: close: Implements the default `_close` implementation used when no specific 
/// implementation is provided at link time.
///  It is called if 'patmosplug_close(int)' is not defined.
```